lvm-operator.yaml

```diff
@@ -1669,7 +1669,7 @@ spec:
           securityContext:
             privileged: true
             allowPrivilegeEscalation: true
-          image: openebs/lvm-driver:1.2.0
+          image: openebs/lvm-driver:1.3.0
```

Comment:
do we need to update openebs.io/version: as well?

Reply:
Good catch! Thanks!